--- NFC/PN532_SPI/PN532_SPI.cpp.orig	2015-12-11 09:38:46.000000000 +0100
+++ NFC/PN532_SPI/PN532_SPI.cpp	2015-12-11 09:39:49.000000000 +0100
@@ -17,7 +17,7 @@
 void PN532_SPI::begin()
 {
     pinMode(_ss, OUTPUT);
-
+#if defined(FALSE)
     _spi->begin();
     _spi->setDataMode(SPI_MODE0);  // PN532 only supports mode0
     _spi->setBitOrder(LSBFIRST);
@@ -27,7 +27,7 @@
     /** DUE spi library does not support SPI_CLOCK_DIV8 macro */
     _spi->setClockDivider(42);             // set clock 2MHz(max: 5MHz)
 #endif
-
+#endif
 }
 
 void PN532_SPI::wakeup()
